src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -557,110 +659,33 @@ private boolean equalsWithNumericDelta(String val, RecordExpectation recordExpec
-        public boolean stringExpectationInCSV = false;
-        public boolean editorStringExpectation = false;
-
-        public RecordExpectation(String fieldName, int intExpectation) {
-            this.fieldName = fieldName;
-            this.expectedFieldType = ExpectedFieldType.INT;
-            this.intExpectation = intExpectation;
-        }
-
-        /**
-         * This extra constructor is a bit hacky in that it is only used for certain records that have
-         * an int type when stored in the database, but a string type when exported to GTFS
-         */
-        public RecordExpectation(String fieldName, int intExpectation, String stringExpectation) {
-            this.fieldName = fieldName;
-            this.expectedFieldType = ExpectedFieldType.INT;
-            this.intExpectation = intExpectation;
-            this.stringExpectation = stringExpectation;
-            this.stringExpectationInCSV = true;
-        }
-
-        /**
-         * This extra constructor is a also hacky in that it is only used for records that have
-         * an int type when stored in the database, and different values in the CSV export depending on
-         * whether or not it is a snapshot from the editor.  Currently this only applies to stop_times.stop_sequence
-         */
-        public RecordExpectation(String fieldName, int intExpectation, String stringExpectation, String editorExpectation) {
-            this.fieldName = fieldName;
-            this.expectedFieldType = ExpectedFieldType.INT;
-            this.intExpectation = intExpectation;
-            this.stringExpectation = stringExpectation;
-            this.stringExpectationInCSV = true;
-            this.editorStringExpectation = true;
-            this.editorExpectation = editorExpectation;
-        }
-
-        public RecordExpectation(String fieldName, String stringExpectation) {
-            this.fieldName = fieldName;
-            this.expectedFieldType = ExpectedFieldType.STRING;
-            this.stringExpectation = stringExpectation;
-        }
-
-        public RecordExpectation(String fieldName, double doubleExpectation, double acceptedDelta) {
-            this.fieldName = fieldName;
-            this.expectedFieldType = ExpectedFieldType.DOUBLE;
-            this.doubleExpectation = doubleExpectation;
-            this.acceptedDelta = acceptedDelta;
-        }
-
-        public String getStringifiedExpectation(boolean fromEditor) {
-            if (fromEditor && editorStringExpectation) return editorExpectation;
-            if (stringExpectationInCSV) return stringExpectation;
-            switch (expectedFieldType) {
-                case DOUBLE:
-                    return String.valueOf(doubleExpectation);
-                case INT:
-                    return String.valueOf(intExpectation);
-                case STRING:
-                    return stringExpectation;
-                default:
-                    return null;
+        if (mismatches != null) {
```

Comment:
I don't think is necessary and request removal.  See discussion in [previous comment](https://github.com/conveyal/gtfs-lib/pull/143/files#r236448259).

Reply:
Continue discussion in https://github.com/conveyal/gtfs-lib/pull/143#discussion_r242725670